src/App.jsx

```diff
@@ -14,57 +15,82 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [chosenItem, setChosenItem] = useState('Jam');
+  const [isSelected, setIsSelectedState] = useState(true);
```

Comment:
I recommend renaming `setIsSelectedState` to `setIsSelected` to maintain naming consistency with the useState functions.
```suggestion
  const [isSelected, setIsSelected] = useState(true);
```